eng/build.ps1

```diff
@@ -38,13 +38,14 @@ foreach ($argument in $PSBoundParameters.Keys)
 {
   switch($argument)
   {
-    "Debug"      { $arguments += " /p:ConfigurationGroup=Debug -configuration Debug" }
-    "Release"    { $arguments += " /p:ConfigurationGroup=Release -configuration Release" }
-    "ExtraArgs"  { $arguments += " " + $ExtraArgs }
-    "Restore"    { $defaultargs = "-restore" }
-    "Test"       { $defaultargs = "-test -warnaserror:0" }
-    "InitTools"  { $defaultargs = "-restore -warnaserror:0" }
-    default      { $arguments += " /p:$argument=$($PSBoundParameters[$argument])" }
+    "Debug"       { $arguments += " /p:ConfigurationGroup=Debug -configuration Debug" }
+    "Release"     { $arguments += " /p:ConfigurationGroup=Release -configuration Release" }
+    "ExtraArgs"   { $arguments += " " + $ExtraArgs }
+    "TargetGroup" { $arguments += " /p:TargetGroup=$($PSBoundParameters[$argument].ToLower())"}
```

Comment:
May not matter, but do you want ToLowerInvariant instead?

Reply:
Good idea.